### ISSUE TITLE
SparsePoly module stores the terms backwards

### ISSUE THREAD

**Issue by tthsqe12** [Nemocas/AbstractAlgebra.jl#564]
in the order opposite the MPoly module. Seems unnecessarily confusing, no?

**Comment by wbhart** [comment]
Yeah that's why the module is not documented and not exposed to the user with a nice interface. It's only for internal use really, and was written before we switched the order elsewhere.

**Comment by lgoettgens** [comment]
it's only internal, so we don't really care
